tests/cross_functional/ui/fusion_access/test_create_filesystem.py

```
@@ -0,0 +1,233 @@
+        Reads the following keys from ENV_DATA (set in your private cluster config):
+            san_iscsi_ip:  IP of the iSCSI target portal, e.g. "192.168.1.100"
+            san_iscsi_iqn: IQN of the iSCSI target, e.g. "iqn.2023-01.com.example:storage"
+        """
+        iscsi_ip = config.ENV_DATA.get("san_iscsi_ip")
+        iscsi_iqn = config.ENV_DATA.get("san_iscsi_iqn")
+        logger.info(
+            f"Starting LUN discovery on all worker nodes "
+            f"(portal: {iscsi_ip}, iqn: {iscsi_iqn})"
+        )
+        ocp_obj = ocp.OCP()
+        ocp_obj.exec_oc_cmd(
```

Ruff (0.15.20)

[error] Function call with `shell=True` parameter identified, security issue

(S604)